src/popup/router/components/AccountSwitcher.vue

```diff
@@ -47,16 +61,25 @@ export default {
       };
     },
     filteredAccounts() {
-      return this.accounts.map((a, index) => ({ ...a, i: index })).filter((a) => a.showed);
+      return [
+        ...this.accounts.map((a, index) => ({ ...a, i: index })).filter((a) => a.showed),
```

Comment:
I'd suggest writing more self explanatory variables - so in that case change `a` to `account`